### ISSUE TITLE
GF bug

### ISSUE THREAD

**Issue by Justice950** [luckydog7/Funkin-android#900]
Why when i put another gf it's still the default gf??

**Comment by NeonNotFound** [comment]
> Why when i put another gf it's still the default gf??

There are a file called gf in shared

**Comment by Justice950** [comment]
> > Why when i put another gf it's still the default gf??
> 
> There are 

> > Why when i put another gf it's still the default gf??
> 
> There are a file called gf in shared

yeah but i changed them bof also the kite assets buy it's the same

**Comment by NeonNotFound** [comment]
> > > Why when i put another gf it's still the default gf??
> > 
> > 
> > There are
> 
> > > Why when i put another gf it's still the default gf??
> > 
> > 
> > There are a file called gf in shared
> 
> yeah but i changed them bof also the kite assets buy it's the same

Oof i recolored gf but it changed

**Comment by VitalieW0WKid** [comment]
> Why when i put another gf it's still the default gf??

In the shared folder it is a folder name "gf"
And you will find the gf assets,are ore assets for her of every action because it needs to be optimised

**Comment by REAL-DoritosMAN** [comment]
You likely missed some files
I installed the gf mod and it works just fine
